Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM php:5.6-apache
+MAINTAINER Guillaume Fillon <guillaume@posteo.de>
+
+
+RUN apt-get update && \
+    apt-get install -y libcurl4-openssl-dev libjpeg62-turbo-dev \
+    libmcrypt-dev libpng12-dev libicu-dev libgmp-dev libsqlite3-dev && \
+    ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/include/gmp.h && \
```

Comment:
I have not tried myself yet, but isn't it possible to have a command a bit more standard, such as 
```
apt-get install php5-curl php5-gd php5-intl php5-json php5-gmp php5-sqlite
```
A bit like in the readme https://github.com/FreshRSS/FreshRSS/blob/beta/README.md#example-of-full-installation-on-linux-debianubuntu